package.json

```diff
@@ -21,7 +21,10 @@
     "react-spinners": "^0.6.1",
     "react-toasts": "^3.0.3",
     "redux": "^4.0.4",
-    "redux-thunk": "^2.3.0"
+    "redux-thunk": "^2.3.0",
+    "reselect": "^4.0.0",
+    "shortid": "^2.2.15",
```

Comment:
We must avoid using such libraries which can be dealt easily at our end, this `shortid` provides random ids which we can take care of ourselves. Extra libraries adds up their footprints on the app bundle size which we would need to optimise at the optimisation stage. Why not handle them now?